Source/LightboxController.swift

```diff
@@ -270,9 +267,7 @@ extension LightboxController: UIScrollViewDelegate {
     if !rotating {
       let pageWidth = collectionSize.width
       let currentPage = Int(floor((collectionView.contentOffset.x - pageWidth / 2) / pageWidth) + 1)
```

Comment:
@RamonGilabert agree with you. A good way is to extract this:
`(collectionView.contentOffset.x - pageWidth / 2) / pageWidth`